Python/instrumentation.c

```diff
@@ -575,6 +580,7 @@ de_instrument(PyCodeObject *code, int i, int event)
         opcode_ptr = &code->_co_monitoring->per_instruction_opcodes[i];
         opcode = *opcode_ptr;
     }
+    assert(opcode != ENTER_EXECUTOR);
```

Comment:
Maybe move this back to just below the `if` block you added? That's effectively where it was before.